CRM/Core/SelectValues.php

```diff
@@ -1072,6 +1072,7 @@ public static function getComponentSelectValues() {
    */
   public static function fieldSerialization() {
     return [
+      CRM_Core_DAO::SERIALIZE_NONE => 'no_serialization',
```

Comment:
Not to nitpick, but I'd prefer for this string to be 
```suggestion
      CRM_Core_DAO::SERIALIZE_NONE => 'none',
```
Because every other string in this array matches up directly with the name of the constant.